tests/TestFramework/PhpUnit/Config/XmlConfigurationHelperTest.php

```diff
@@ -232,6 +233,77 @@ public function test_it_removes_existing_printers(): void
         );
     }
 
+    public function test_it_validates_xml_by_root_node(): void
+    {
+        $this->expectException(InvalidPhpUnitXmlConfigException::class);
+        $this->expectExceptionMessage('phpunit.xml does not contain a valid PHPUnit configuration.');
```

Comment:
How about moving this block to right before performing the action instead?

For reference, also see https://thephp.cc/news/2016/02/questioning-phpunit-best-practices.

Reply:
Done